src/test/kotlin/com/terraformation/backend/email/EmailNotificationServiceTest.kt

```diff
@@ -1348,6 +1484,25 @@ internal class EmailNotificationServiceTest {
     }
   }
 
+  private fun assertNoMessageSent() {
```

Comment:
This method is to give more visibility to what emails were sent, instead of a generic "Expected to be empty but was MimeMessage object"